.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+commands:
+  before_install:
+    steps:
+      - run: |
+          dnf install make -y
+
+  install:
+    steps:
+      - run:
+          name : list_folder
+          command : ls
+      - run:
+          name : install_cmd
+          command: make install
+  
+  before_script:
+    steps:
+      - run: |
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" tls $NUO_SET_TLS
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" start
+
+
+  script:
+    steps:
+      - run:
+          name: script_cmd
+          command: make test NUODB_HOME="/opt/nuodb"
+  
+  after_failure:
```

Comment:
It's better to replace this with `store_artifacts` step. See https://circleci.com/docs/artifacts/ for more details.